renovate.json5

```diff
@@ -11,11 +11,12 @@
       "automerge": true,
     },
     {
-      "description": "Compose compiler is tightly coupled to Kotlin version.",
-      "groupName": "Kotlin and Compose",
+      "description": "Compiler plugins are tightly coupled to Kotlin version.",
```

Comment:
For reference Compose prefix was removed here: https://github.com/cashapp/paparazzi/commit/4c9a34bebef170637f35f7e04873a7a42ae021af#diff-31b1aa2b93afe11792d84c6cb5057a064b5a3294fa0caab92d7570ea604f759fR17